dojo/tools/openvas/xml_parser.py

```diff
@@ -38,10 +40,12 @@ def get_findings(self, filename, test):
 
             finding = Finding(
                 title=str(title),
+                test=test,
```

Comment:
Seems like a bugfix, likely best in a separate PR.

Reply:
here #11464